src/RoverSelection/RoverSelection.tsx

```diff
@@ -1,6 +1,124 @@
 function RoverSelection() {
     return (
-      <div>Rover selection page</div>
+      <div>
+       <h1> Rover selection page </h1>
+        <div className='CuriosityRover'>  
+          <h2> Curiosity Rover:</h2>
+          <img src='https://upload.wikimedia.org/wikipedia/commons/thumb/f/f3/Curiosity_Self-Portrait_at_%27Big_Sky%27_Drilling_Site.jpg/520px-Curiosity_Self-Portrait_at_%27Big_Sky%27_Drilling_Site.jpg' width='200px' height='200px' alt='Image of Curiosity Rover' />
+          <h3> Stats</h3>
+          <table className='CuriosityTable'>
```

Comment:
Could we make these tables of data into react components that we can reuse for each one? We can then define some props that can be passed in that holds all the relevant information and then pass in the relevant information to each component for the 3 rovers. It might be worth defining an interface that describes what the structure of the data looks like and hard coding the data below in a separate data file. Feel free to grab me to go over this at some point :)

Reply:
done ! :)